Kernel/Graphics/Intel/DisplayConnectorGroup.h

```diff
@@ -41,6 +41,11 @@ class IntelDisplayConnectorGroup : public RefCounted<IntelDisplayConnectorGroup>
 private:
     AK_TYPEDEF_DISTINCT_ORDERED_ID(size_t, RegisterOffset);
 
+    enum class AnalogOutputRegisterOffset {
```

Comment:
` Kernel: Rename GlobalGenerationRegister => AnalogOutputRegisterOffset ` feels like it could be squashed into the initial commit that creates the DisplayConnectorGroup.